### ISSUE TITLE
Issues generating figures using the command line

### ISSUE THREAD

**Issue by sheinasim** [blobtoolkit/blobtoolkit#165]
Hello!

I installed this recent version of blobtoolkit on our HPC using the conda and pip instructions but when I run this command:

```
blobtools view --view snail --format png --out blobdir_figs blobdir
```

I get this error:

```
FileNotFoundError: [Errno 2] No such file or directory: 'Xvfb'
```

I don't think I can install Xvfb on our clusters, but the help desk suggested that I try to run it using docker (which we actually run using singularity).

So I am now using this command:

```
singularity exec blobtoolkit_latest.sif blobtools view --view snail --format png --out blobdir_figs blobdir
```

And it doesn't result in any errors, but it also doesn't write any files. It just writes this and gets stuck:

```
Initializing viewer
Loading http://localhost:8001/view/primary.purged_noec/dataset/primary.purged_noec/snail?staticThreshold=Infinity&nohitThreshold=Infinity&plotGraphics=svg
Fetching primary.purged_noec.snail.png
 - waiting for element snail_save_png
```

If I could create these figures through the command line, I would be a hero at work. 

Any suggestions for how I can resolve this?

Thanks so much!
Sheina

**Comment by rjchallis** [comment]
I haven't been able to reproduce this kind of issue so not sure how to fix it directly, but I'm actually working on a command line plotting option that doesn't need to run the viewer at all to make these kinds of issues go away. 

It won't be fully integrated into blob toolkit for a while but I should have a working version of the snail plot ready by the end of the week that you'll be able to try out.

**Comment by sheinasim** [comment]
Hi Richard!

I figured out the issue, it works great now! (I was trying to create files from a blobdir created by an earlier version of blobtools).

So I created it with the current version and was able to run `blobtools view` in the conda environment and docker/singularity container.

It would still be great to be able to change the figure to say "contig" instead of "scaffold" when appropriate, but that's just me being greedy.

Haha.

Thanks!
Sheina
